src/components/GUI/proofEditor/issuePositionInformation.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
-import Position from "../../issueHandling/position";
-import styles from "./proofEditor.module.scss";
+import Position from "../../../issueHandling/position";
+import styles from "../../CSS/proofEditor.module.scss";
```

Comment:
style files are usually kept next to their component's `tsx` files because they are so tightly coupled. Is there a reason to move all `scss` files into a separate `CSS` folder?